axelrod/tests/strategies/test_resurrection.py

```diff
@@ -33,3 +33,29 @@ def test_strategy(self):
         
         # Check for four defections and a cooperation by opponent
         self.responses_test([D], [C] * 5, [D, D, D, D, D, C])
+        
+class TestDoubleResurrection(TestPlayer):
+
+    name = "DoubleResurrection"
+    player = axelrod.DoubleResurrection
+    expected_classifier = {
+        'memory_depth': 1,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        # Starts by Cooperating
+        self.first_play_test(C)
```

Comment:
Would you be able to re write these tests using the new framework? 

I know you started writing this before the change but we'd appreciate it :)

http://axelrod.readthedocs.io/en/latest/tutorials/contributing/strategy/writing_test_for_the_new_strategy.html